builders/toolkit/integrations/wallets/rainbowkit.md

```diff
@@ -0,0 +1,314 @@
+
+### Checking Prerequisites {: #checking-prerequisites }
+
+Before proceeding, ensure the following prerequisites are met:
+
+- An existing dApp built with [React](https://react.dev/){target=\_blank} and manual setup will be used to connect to a wallet via RainbowKit.
+
+- The [RainbowKit examples repository](https://github.com/rainbow-me/rainbowkit/tree/main/examples){target=\_blank} includes templates for multiple React frameworks
+    
+ - To follow this guide, visit [Next.js](https://nextjs.org/docs){target=\_blank} and follow the **Automatic Installation** instructions, selecting Typescript and the App Router options during setup 
+
+- The wallet being used must support custom networks, as the Dancebox appchain will be added manually.
+
+- A WalletConnect `projectId` - every dApp relying on WalletConnect is required to have an associated `projectId`. It is free to create an account,  and an ID can be generated instantly.
+
+    To obtain a WalletConnect `projectId`:
+
+    1. Visit [WalletConnect Cloud](https://cloud.walletconnect.com/){target=\_blank}
+    2. On the **Projects** page, select **Create** 
+    3. Add the project information (leaving the **Homepage URL** blank if the dApp is not deployed)
+    4. Select the **AppKit** SDK
+    5. Select a coding environment or platform (React is used in this guide)
+    6. Locate the `projectId` in the left menu. or find it in the **Get started** code snippet of WalletConnect Quickstart.
+ 
+### Getting Started {: #getting-started }
+
+Ensure that the project is in the root directory before proceeding, then install RainbowKit and its peer dependencies:
+
+=== "npm"
+
+    ```bash
+    npm install @rainbow-me/rainbowkit wagmi viem@2.x @tanstack/react-query
+    ```
+
+=== "pnpm"
+
+    ```bash
+    pnpm install @rainbow-me/rainbowkit wagmi viem@2.x @tanstack/react-query
+    ```
+
+=== "yarn"
+
+    ```bash
+    yarn add @rainbow-me/rainbowkit wagmi viem@2.x @tanstack/react-query
+    ```
+These packages provide the core functionality for wallet connections (RainbowKit), Ethereum interactions (wagmi and viem), and state management (TanStack Query).
+
+Next, start the development server to create a local dApp instance:
+
+=== "npm"
+
+    ```bash
+    npm run dev
+    ```
+
+=== "pnpm"
+
+    ```bash
+    pnpm run dev
+    ```
+
+=== "yarn"
+
+    ```bash
+    yarn dev
+    ```
+
+After starting the server, open [http://localhost:3000](http://localhost:3000){target=\_blank},  to view the Next.js application in the browser.
+
+To test the RainbowKit connection, the MetaMask app can be used. Ensure that the Dancebox appchain is connected in the MetaMask wallet.
```

Comment:
Change every "Dancebox appchain" to "Tansso demo EVM appchain" in every occurrence